tests/spec/fs.lstat.spec.js

```diff
@@ -47,3 +47,29 @@ describe('fs.lstat', function() {
     });
   });
 });
+
+describe('fs.promises.lstat', function() {
```

Comment:
Just nitpick there, you can wirte a function like () => instead of function(). The arrow function expression has a shorter syntax than a function expression.

Reply:
Good suggestion @xwang345 - in a case like this, I'd vouch to not incorporate arrow functions since other tests don't use that syntax. Consistency across code goes a long way!